temboardagent/configuration.py

```diff
@@ -1,22 +1,29 @@
+import logging
+
 try:
     import configparser
 except ImportError:
     import ConfigParser as configparser
 
-import os
+import os.path
```

Comment:
hmm, actually, os.path is a submodule of `os`. It happen to be imported already, even in a simple `python` interactive shell. But it's not theorically right. See:

```
>>> import logging
>>> logging.handlers
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'handlers'
>>> import logging.handlers
>>> logging.handlers
<module 'logging.handlers' from '/usr/lib/python2.7/logging/handlers.pyc'>
>>>
```

The submodules `handlers` does not exists until it is explicitly imported. Thus, if use the `os.path` module, i import it explicitly.

The other point is that `import X.Y` creates actually *one* variable in module scope, named `X`. `Y` is an attribute of `X` pointing to the submodule. This mean that importing `X.Y` is equivalent to `import X, X.Y`. This is why i *replace* `import os` with `import os.path`.

But yes, that's useless. :-) I remember a case where `os` was imported but *not* `os.path`, but i can't reproduce it.